web/src/components/Container.tsx

```diff
@@ -6,3 +6,12 @@ export const Container: React.FunctionComponent<ContainerProps> = (
 ) => {
   return <C maxWidth="md" {...props} />
 }
+
+type OptionalContainerProps = ContainerProps & { noContainer?: boolean }
+export const OptionalContainer = (props: OptionalContainerProps) => {
```

Comment:
Why not only have the one container? Why the two different classes? Almost seems like this class shouldn't if a if (props.noContainer) ?